.github/workflows/build.yaml

```diff
@@ -60,5 +68,6 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: ./${{ matrix.directory }}
-          push: ${{ env.PUSH_IMAGE }}
+          push: ${{ github.event_name == 'push' && true || false }}
```

Comment:
Can this just be ${{ github.event_name == 'push' }}?

Reply:
That did not work, which is why I used the fromJSON originally (https://github.com/actions/runner/issues/1483). But Aaron's suggestion works.